docs/1.12/03-Tutorials2/06-Deploy-Prisma-Servers/03-Digital-Ocean-(manual).md

```diff
@@ -56,97 +60,129 @@ curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+```sh
+docker-compose up -d
+```
 
-#### Copy cluster configuration from server
+This will fetch the docker images for both `prisma` and `mysql`. To verify that docker is running these images, run:
 
-On your digital ocean droplet, copy the clusterSecret for the `local` cluster:
+```sh
+docker ps
+```
 
 ```sh
-> cat ~/.prisma/config.yml
-clusters:
-  local:
-    host: 'http://localhost:4466'
-    clusterSecret: "-----BEGIN RSA PRIVATE KEY----- [ long key omitted ] -----END RSA PRIVATE KEY-----\r\n"
+CONTAINER ID        IMAGE                       COMMAND                  CREATED             STATUS              PORTS                    NAMES
+24f4dd6222b1        prismagraphql/prisma:1.12   "/bin/sh -c /app/sta…"   15 seconds ago      Up 1 second         0.0.0.0:4466->4466/tcp   root_prisma_1
+d8cc3a393a9f        mysql:5.7                   "docker-entrypoint.s…"   15 seconds ago      Up 13 seconds       3306/tcp                 root_mysql_1
 ```
 
-#### Add new cluster in local configuration
+## 5. Deploy the Service
+
+Now that your infrastructure is running via docker, we will deploy the prisma service.
+
+For us to enable deploys from our local machines, open up `~/.prisma/config.yml` in your favorite editor on YOUR machine.
 
-On your local development machine open `~/.prisma/config.yml` in your favourite editor. It is likely that you already have one or more clusters defined in this file. Add another cluster and give it a descriptive name. For example `do-cluster`:
+It is likely that you already have one or more services defined in this file. Add another service and give it a descriptive name. For example do-cluster:
 
 ```yml
 clusters:
   local:
     host: 'http://localhost:4466'
-    clusterSecret: "-----BEGIN RSA PRIVATE KEY----- [ long key omitted ] -----END RSA PRIVATE KEY-----\r\n"
   do-cluster:
     host: 'http://37.139.15.166:4466'
-    clusterSecret: "-----BEGIN RSA PRIVATE KEY----- [ long key omitted ] -----END RSA PRIVATE KEY-----\r\n"
 ```
 
-Change the host to use the ip of your Digital Ocean droplet.
+Next we are going to bootstrap a new project via `prisma init`. On your local machine input:
 
-#### Deploy a service
+```sh
+prisma init hello-world
+```
 
-On your local machine, verify that the cluster configuration is being picked up correctly:
+Following the interactive prompt, select `Use other server`. Enter the ip of the digital ocean droplet, and follow the prompt for adding the management secret.
+
+This command should output:
 
 ```sh
-prisma cluster list
-```
+Created 3 new files:
 
-The output should include your newly added cluster.
+prisma.yml           Prisma service definition
+datamodel.graphql    GraphQL SDL-based datamodel (foundation for database)
+.env                 Env file including PRISMA_API_MANAGEMENT_SECRET
+```
 
-Now you can create a new service and deploy it to the cluster:
+Next `cd` into the `hello-world` directory generated from the `prisma init` command and run:
```

Comment:
I know it's tempting to write smth like "Next `cd` into the `hello-world` directory", but I want to be _extremely_ explicit about all steps. So let's switch that to: 

Next navigate into the `hello-world` directory that was generated from `prisma init` and deploy your Prisma service with the following commands:

```
cd hello-world
prisma deploy
```